src/main/java/main/mainApp.java

```diff
@@ -0,0 +1,57 @@
+package main;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import model.Human;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Stream;
+
+public class mainApp {
+
+    public static void main(String[] args) throws IOException {
+        Map<Human, Human> offPrint = new HashMap<>();
+        File humanfile = new File("src/main/resources/HumanList.json");
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<Human> humanList = objectMapper.readValue(humanfile, new TypeReference<>() {
+        });
+        List<Human> santaList = objectMapper.readValue(humanfile, new TypeReference<>() {
+        });
+        System.out.println("Number of humans " + humanList.size());
+
+        if (humanList.size() % 2 != 0) {
+            System.out.println("HumanList.json size is odd, we need one more human to run the Secret Santa Factory ");
+            return;
+        }
+
+        santaList
+                .forEach(santa -> {
+                    Stream<Human> humanStream = humanList.stream();
+                    Collections.shuffle(humanList);
```

Comment:
Why shuffle the humanList here if you declare a stream using it at the previous line ?